examples/checkpoint/run_deepseek_megatron_ckpt.sh

```diff
@@ -0,0 +1,91 @@
+set -x
```

Comment:
I guess we can turn on the checkpoint functionality to old examples to avoid adding too many examples

Reply:
I guess we can keep the current changes